src/js/resize.js

```diff
@@ -0,0 +1,22 @@
+import Split from 'split.js'
+
+const PAGE_WIDTH = 960
+
+const isSplited = () => document.getElementsByClassName('gutter').length > 0
+const isBigScreen = () => window.innerWidth > PAGE_WIDTH
+const main = document.getElementById('main')
+const gutter = () => document.getElementsByClassName('gutter')[0]
+
+const makeGutter = () => {Split(['#left_pane', '#right_pane'], {
+    gutterSize: 8,
+    cursor: 'col-resize'
+  })}
+
+export default function resize() {
+    if (isBigScreen() && !isSplited()) {
+        makeGutter()
+    } else if (!isBigScreen() && isSplited()) {
+        main.removeChild(gutter())
```

Comment:
There is method `.destroy()` that should perform clean-up.

We could use it and create new instance in its place?

Reply:
I am aware of that. But it doesn't work properly. It brakes layout anyway and causes an error.